packages/components/src/components/common/toggleGroup/toggleGroup.css.ts

```diff
@@ -6,7 +6,6 @@ export const toggleGroup = recipe({
     display: 'inline-flex',
     alignItems: 'center',
     gap: vars.size.s2,
-    padding: '10px',
     backgroundColor: vars.color.background,
     borderRadius: vars.border.r2
```

Comment:
maybe also useless?

Reply:
you are right, I will remove these in another PR. However, I no longer sit there as a reviewer because the change is too small.